.github/workflows/update-drci-comments.yml

```diff
@@ -0,0 +1,22 @@
+name: Update Dr. CI comments
+
+on:
+  schedule:
+    # Run every 15 minutes
+    - cron: "*/15 * * * *"
+  # Have the ability to trigger this job manually through the API (temporary)
+  workflow_dispatch:  
+
+jobs:
+  update-drci-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Retrieve rockset query results and update Dr. CI comments
+        run: |
+          curl --request POST \
+          --url 'https://www.torch-ci.com/api/drci/drci' \
+          --header 'Authorization: ${{ secrets.GITHUB_TOKEN }}'
+        # not sure what authorization tokens i need
```

Comment:
The authorization is to make sure that people can't just go to this URL and spam our API calls --> you would want to create some random token + put it in Vercel and also as a secret in this repo, similar to my disable bot GHA.